src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/hiddenfield/Example1.java

```diff
@@ -0,0 +1,29 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="HiddenField"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.hiddenfield;
+
+// xdoc section -- start
+public class Example1 {
+
+  private String field;
+  private String testField;
+
+  public Example1(String testField) { // violation
+  }
+  public void method(String param) { // OK
+    String field = param; // violation, 'field' variable hides 'field' field
+  }
+  public void setTestField(String testField) { // violation
```

Comment:
Please remove "public" modifier from all method that do not have to o have it to minimize code

Reply:
Done